.travis.yml

```diff
@@ -54,7 +23,5 @@ matrix:
       env: DB=mariadb10
     - rvm: 2.0.0
       env: DB=mysql57
-    - rvm: rbx-2
-      env: RBXOPT=-Xgc.honor_start=true
```

Comment:
I needed this environment variable to get deterministic `GC.start` behavior on Rubinius. Is it no longer needed, or worked around in some other way?